packages/framework/fluid-framework/package.json

```diff
@@ -76,8 +76,10 @@
       "ClassDeclaration_SharedDirectory": {"forwardCompat": false},
       "ClassDeclaration_SharedString": {"forwardCompat": false},
       "ClassDeclaration_FluidContainer": {"forwardCompat": false, "backCompat": false},
-      "InterfaceDeclaration_IFluidContainer": {"backCompat": false}
-    },
-    "disabled": true
```

Comment:
?

Reply:
This also confused me 😅. I'm not sure exactly what happened, but when I merged the latest changes from the next branch, I noticed that there was a merge conflict here with `"disabled": true`. I checked both this branch and the next branch and it seemed to not exist on either. Either way I think we should be fine now